src/ti_comm/src/README.md

```diff
@@ -0,0 +1,10 @@
+How to Build Libserialport
```

Comment:
Remove this file as a whole if you agree with the suggestion to move its contents t the top-level README